eng/scripts/Language-Settings.ps1

```diff
@@ -589,17 +591,29 @@ function Import-Dev-Cert-python
   python $pathToScript
 }
 
+# Defined in common.ps1 as:
+# $ValidateDocsMsPackagesFn = "Validate-${Language}-DocMsPackages"
 function Validate-Python-DocMsPackages ($PackageInfo, $PackageInfos, $PackageSourceOverride, $DocValidationImageId)
 {
   # While eng/common/scripts/Update-DocsMsMetadata.ps1 is still passing a single packageInfo, process as a batch
   if (!$PackageInfos) {
     $PackageInfos =  @($PackageInfo)
   }
 
+  $allSucceeded = $true
```

Comment:
Is this function also loops over a set of packageInfos? Any reason to not pass all of them to this function at https://github.com/Azure/azure-sdk-tools/commit/b3643479716f5499a97a82602f2ca6d81c0b5422#diff-f713405e8e202dd6440aa9ba91f509c16b6ea926815ff2adee5fcdce960996f6R242?

Reply:
One could do that but then the domain-specific message indicating that a package failed validation would need to be written in as well (and in other `Language-Settings.ps1` files in other repos). I tried to put that logging outside of the validation function in the common place where the function is called instead of making the validation function responsible for describing which packages succeeded or failed validation.